.claude/CLAUDE.md

```
@@ -1,1319 +1,78 @@
-5. **Schemas in `schemas/` folder** - separate from core, organized by module
-6. **Models in `models/` folder** - separate from core, organized by module
-7. **Exceptions in Core** - each feature has its own exceptions file
-
----
-
-## Layer Architecture
-
-### Standard Module Structure
+## Architecture at a glance
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)